### PR TITLE
Sync with `cumulus/parachain-template`

### PR DESCRIPTION
**Pull request by cmichi**
It's just this commit: https://github.com/paritytech/cumulus/commit/77491879cd5af1717c459e7b980b7e9a270617c6#diff-ec2df2e97e27873c3143845320b3b5175fcf92bc76a103f4a1755745cb3cf049.

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -593,6 +593,8 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = ();
 	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+	type ControllerOrigin = EnsureRoot<AccountId>;
```
Comment (cmichi) [inline runtime/src/lib.rs:596]: @riusricardo I'm not sure when we should use `EnsureRoot` and when we should use `EnsureOneOf<EnsureRoot<AccountId>, EnsureXcm<IsMajorityOfBody<RelayLocation, ExecutiveBody>>>;` (so also allowing the relay chain council). Wdyt?

Comment (riusricardo) [reply runtime/src/lib.rs:596]: This is just a security measure to disable XCM. But we don't have sudo in Canvas so this functionality will not work unless we add the EnsureXcm to handle it from the relay chain.

This opens the question of adding a second runtime with sudo for a much easier and faster development.

Comment (HCastano) [reply runtime/src/lib.rs:596]: > This opens the question of adding a second runtime with sudo for a much easier and faster development.

There's gotta be a better way to tune this one setting than to add the overhead of a second runtime 😄

Comment (athei) [reply runtime/src/lib.rs:596]: @riusricardo so which of both do we want. I think for common good we want `EnsureOneOf<EnsureRoot<AccountId>, EnsureXcm<IsMajorityOfBody<RelayLocation, ExecutiveBody>>>;`, right?

Comment (cmichi) [reply runtime/src/lib.rs]: Alright, I've updated it. Could someone approve the PR plz?

File: README.md [hunk 2]
```diff
@@ -74,7 +75,7 @@ Once you have ROC on Canvas you can deploy a contract as you would normally.
 If you're unsure about this, our [guided tutorial](https://docs.substrate.io/tutorials/v3/ink-workshop/pt1/) 
 will clarify that for you in no time.
 
-As an endowment for our examples contracts 1-2 ROC tokens should do.
+As an endowment for our example contracts 1-2 ROC tokens should do.
```
Comment (athei) [inline README.md:78]:
```suggestion
```

No longer a thing.

**Review by athei — APPROVED** [review]

**Pull request merged**